packages/components/psammead-image/src/testHelpers/stories.jsx

```diff
@@ -69,5 +69,20 @@ const stories = (
       ),
       { notes },
     );
+  if (Component.name === 'Img') {
+    imageStories.add(
+      'image without width',
+      () => (
+        <Component
+          {...getProps(landscape, includeHeight)}
+          {...additionalProps}
+          width={190}
```

Comment:
If this new story is for `images without width` why are we passing a `width` prop?

Reply:
Still in progress. i was just testing out some stuff.